cunumeric/eager.py

```diff
@@ -541,6 +541,15 @@ def flip(self, rhs: Any, axes: Union[None, int, tuple[int, ...]]) -> None:
         else:
             self.array = np.flip(rhs.array, axes)
 
+    def broadcast_to(self, shape: NdShape) -> NumPyThunk:
```

Comment:
this code doesn't make sense at all. why is an eager array always converted to a deferred array here? can't this be implemented using `numpy.broadcast_to`?

Reply:
As far as I remember, this was intended to handle a case when eager and deferred array are used for a operation. In this case, a broadcasted eager array should be changed to a corresponding deferred array, which doesn't work w/ `deferred.broadcast_to`. So, I changed this code to convert eager array to `deferred` when `broadcast_to`is called. I couldn't find a way to handle this case on my own w/o this change.